hivelink-core/src/main/java/org/apache/iceberg/hivelink/core/LegacyHiveTableOperations.java

```diff
@@ -68,6 +71,9 @@
 public class LegacyHiveTableOperations extends BaseMetastoreTableOperations {
 
   private static final Logger LOG = LoggerFactory.getLogger(LegacyHiveTableOperations.class);
+  static final long INITIAL_SEQUENCE_NUMBER = 0;
```

Comment:
why not declare it as `private` for these three constant variables?

Reply:
Thanks for pointing it out, modified them.